tests/plugins/test_resolve_remote_source.py

```diff
@@ -116,7 +116,7 @@ def mock_repo_config(workflow, data=None):
     workflow.source.config = SourceConfig(str(workflow._tmpdir))
 
 
-def mock_cachito_api(workflow, user=KOJI_TASK_OWNER, source_request=None):
+def mock_cachito_api(workflow, user=KOJI_TASK_OWNER, source_request=None, dep_rep=None):
```

Comment:
We have lots of names for the same value: dep_rep, dependency_replacements, dep_replacement. Could we give them all the same name?

Reply:
Sure! Fixed.